apps/old/widget/components/buttons/UserDropdown.jsx

```diff
@@ -179,7 +179,7 @@ return (
           className="dropdown-item"
           type="button"
           to={href({
-            widgetSrc: "${config_index}",
+            widgetSrc: "${alias_new}/widget/Index",
```

Comment:
I hope this doesn't break anything for the existing widgets which use this.

Reply:
the config_index alias was giving a warning because it was not defined.